src/client/proxy/handlers/common.ts

```diff
@@ -235,7 +235,25 @@ const attachCommonHandler = (server: FastifyInstance, ctx: any) => {
     const wsHandler = ({socket}: SocketStream, {hostname}: FastifyRequest) =>
         void new ZProxySocketController(ctx, socket, server.websocketServer, hostname);
 
-    server.route({method: 'GET', url: '*', handler, wsHandler});
+    // Handle websocket requests.
+    server.route({
+        method: 'GET',
+        url: '/ws',
```

Comment:
I was under impression we decided to not use specific paths for ws connection?

Reply:
Yes, but as I noted in the description, when I started implementing it, I wanted to add the authorization as a _preValidation_ hook, and if we use the same handler for everythng, the _preValidation_ is going to affect everything and not just websocket.

But I can change it and find a different way to handle the authorization check, let me know.